src/asynch/ws.rs

```diff
@@ -146,23 +192,33 @@ mod edge_net_impl {
     }
 }
 
-#[cfg(feature = "embedded-svc")]
+#[cfg(any(feature = "embedded-svc", feature = "embedded-svc-prost"))]
```

Comment:
Revert. Also and again: pleae note that ideally you should conditionalize the `edge-net` WS sender/receiver impls with `prost`, not just the `embedded-svc` ones.